File: api/javascript/IReportService.ts
/// <reference path="./YadelModel" />
export interface IReportService {
	delete_dag(request: org.roylance.yadel.UIRequest, onSuccess:(response: org.roylance.yadel.UIResponse)=>void, onError:(response:any)=>void)
	current(request: org.roylance.yadel.UIRequest, onSuccess:(response: org.roylance.yadel.UIResponse)=>void, onError:(response:any)=>void)
}

File: sapi/src/main/java/org/roylance/yadel/services/IReportService.kt
package org.roylance.yadel.services

interface IReportService {
	fun delete_dag(request: org.roylance.yadel.YadelReport.UIRequest): org.roylance.yadel.YadelReport.UIResponse
	fun current(request: org.roylance.yadel.YadelReport.UIRequest): org.roylance.yadel.YadelReport.UIResponse
}